app/src/main/java/com/infomaniak/drive/data/models/File.kt

```diff
@@ -198,16 +201,16 @@ open class File(
 
     fun isPDF() = getFileType() != ExtensionType.PDF
 
-    fun onlyOfficeUrl() = if (externalShareLinkUuid.isBlank()) {
-        "${BuildConfig.AUTOLOG_URL}?url=" + ApiRoutes.showOffice(this)
-    } else {
+    fun onlyOfficeUrl() = if (isPublicShared()) {
```

Comment:
I know those existing names aren't the best. 
Instead of renaming some here, what I would like to do is make a different PR that directly change all these functions to property. That would solve the probleme and actualize the code base at the same time